sklearn/neighbors/_nearest_centroid.py

```diff
@@ -60,6 +61,10 @@ class NearestCentroid(ClassifierMixin, BaseEstimator):
     shrink_threshold : float, default=None
         Threshold for shrinking centroids to remove features.
 
+    priors : array-like of shape (n_classes,), default=None
```

Comment:
I would probably expect to have 2 strings: `{"uniform", "empirical"}` and the default could be `"empirical"` instead of `None`. This is more explicit.

Reply:
With the current setup, the users can specify any priors as they desired and with your suggestion, the priors are limited to just two options so mine has more flexibility. What do you think?

Reply:
I meant to add uniform and empirical in addition of the array-like. I only don't want to have the `None` option that is not explicit what it is doing. I agree that we need flexibility with the current array-like implementation